pkg/controller/controller_test.go

```diff
@@ -21,7 +21,7 @@ type operation int
 
 const (
 	opAdd operation = iota
-	opUpdate
+	opUpdate operation = iota
```

Comment:
You only need to declare iota on the first const,  the complier will increment and assign the value to the following constants.